ert_shared/status/tracker/evaluator.py

```diff
@@ -49,13 +54,16 @@ def __init__(
         self._work_queue = queue.Queue()
 
         self._drainer_thread = threading.Thread(
-            target=self._drain_monitor, name="DrainerThread"
+            target=self._drain_monitor,
+            name="DrainerThread",
+            args=(next_ensemble_evaluator_wait_time,),
```

Comment:
Is there a special reason `next_ensemble_evaluator_wait_time` is passed as an argument to `_drain_monitor` and not made available inside the function similar to `token` or `cert`?

Reply:
Fixed!